versions.tf

```
@@ -1,3 +1,3 @@
 terraform {
-  required_version = ">= 0.12.0, < 0.14.0"
+  required_version = ">= 0.12.0, < 0.14.1"
```

```suggestion
  required_version = ">= 0.12.0"
```

We don't use ceiling for the versions anymore.

Please update the example as well https://github.com/cloudposse/terraform-null-label/blob/master/examples/complete/versions.tf

Please also run the following commands from the repo root:

```
make init
make github/init
make readme
```

Thank you @kiddom-kq

---

Not 100% why, but one of the make targets did not install `gomplate`, i had to `brew install gomplate` and then all three targets ran w/o error.